src/cluster.c

```diff
@@ -328,7 +328,7 @@ int clusterLoadConfig(char *filename) {
             clusterAddNode(n);
         }
         /* Format for the node address and auxiliary argument information:
-         * ip:port[@cport][,hostname[,aux=val]*] */
+	 * ip:port[@cport][,hostname][nodename=nodenameval][,aux=val]*] */
```

Comment:
```suggestion
         * ip:port[@cport][,hostname][,aux=val]*] */
```
Is there a reason that nodename can't be an aux field? The idea was that aux fields would be generic and we could add more later, it should remove some of this parsing code ideally.

Reply:
Addressed